### DISCUSSION TITLE
[Dashboard] Refine Dashboard Capabilities: Add descriptions for all workloads as traits

### DISCUSSION THREAD

**Discussion by zzxwill** [kubevela/kubevela#821] [General] [answered]
- Add descriptions for all workloads as traits

![image](https://user-images.githubusercontent.com/2805315/103117691-8bdd8c80-46a6-11eb-8973-001ca75429a0.png)
![image](https://user-images.githubusercontent.com/2805315/103117695-9566f480-46a6-11eb-853f-11d3add746c0.png)

**Comment by sunny0826** [comment, marked as answer]
API of `/api/workloads`  has no `descriptions` field

**Reply by zzxwill** [reply]
Okay, let me fix it.

**Reply by zzxwill** [reply]
Fixed. https://github.com/oam-dev/kubevela/pull/824
